test/dtransform.c

```diff
@@ -124,7 +124,7 @@ const int transformData[ROWS][COLS] = {{36, 31, 25, 19, 13, 7, 1, 5, 11, 16, 22,
         const char *f_to_c = "(5/9.0)*(x-32)";                                                               \
         /* utrans is a transform for unsigned types: no negative numbers involved and results are < 255 to   \
          * fit into uchar */                                                                                 \
```

Comment:
It seems from the comments in issue #1381 that the reason for the change is that where this test was failing, char is a signed type, and the change was made to avoid numbers outside the range -128 to 127, as in 100 + 36 and 100+ 31.  The function change avoids that.  The comments don't agree - do they need to be revised?

Reply:
Comment is confusing because the type is just char - maybe just delete the comment.
OR indicate that it is char and the range is -128 to +127

Reply:
I guess utrans might not be signed, but depending on the particular test, x is sometimes signed, so for some compilers the calculation is effectively restricted to values that fit in the signed char range -128 to 127?  As you say, maybe delete the comment or state the narrower range in case the input is signed.

Reply:
Revised comments